docs/help/contributor/style-guide/style-and-formatting.md

```diff
@@ -1,11 +1,11 @@
 # Formatting standard and conventions
 
 ## Documenting Knative API Objects
-> Any Knative API Objects ought to use Title case.
+>  Use title case for Knative API Objects.
 
 |:white_check_mark: Correct                  |:no_entry: Incorrect
 |--------------------|-----
-|Channels | `Channels`
+|Channels | Channels
```

Comment:
Are these supposed to be the same? Should the right hand side be "channels" maybe? Or maybe an example with two words, like TrafficSpec (or Traffic Spec??) vs trafficSpec

Reply:
Sorry, yes. Daft error; I was probably rushing.